airbyte-integrations/connectors/source-instagram/integration_tests/spec.json

```diff
@@ -18,6 +18,14 @@
         "description": "The value of the access token generated. See the <a href=\"https://docs.airbyte.com/integrations/sources/instagram\">docs</a> for more information",
         "airbyte_secret": true,
         "type": "string"
+      },
+      "ig_user_ids": {
+        "title": "Instagram Business Account ID(s)",
+        "type": "string",
+        "description": "Comma separated list of Instagram Business Account IDs. Each account ID must be specified as a 15-digit number without dashes. See the <a href=\"https://docs.airbyte.com/integrations/sources/instagram\">docs</a> for more information",
```

Comment:
Question: Since the Account ID is 17 digits long, why do you describe it as 15 digit?

Reply:
Sorry, my bad! fixed it.